src/components/Footer/index.js

```diff
@@ -0,0 +1 @@
+export { default as Footer } from "./Footer"
```

Comment:
I guess you can write just `export Footer from "./Footer"` and omit `{ default as Footer }`

Reply:
I'm afraid `export Footer from "./Footer` isn't syntactically correct, in my case I followed @frinyvonnick's article https://alligator.io/react/index-js-public-interfaces.